lib/tools/adb-emu-commands.js

```diff
@@ -38,9 +38,15 @@ emuMethods.fingerprint = async function (fingerprintId, udid = undefined) {
   if (udid) {
     await this.setDeviceId(udid);
   }
+  this.resetTelnetAuthToken();
   await this.adbExec(['emu', 'finger', 'touch', fingerprintId]);
 };
 
+emuMethods.rotate = async function () {
```

Comment:
does this call happen in appium-android-driver? I'm just trying to build the full picture. 

And there is also one more thing, which bothers me a bit - we always use 'adb emu' call without any additional arguments. Will it work as expected in case there are multiple simulators running? Probably, one has to specify it explicitly to which simulator the command is going to be sent (like -s option for adb with real device)?